build.gradle

```diff
@@ -212,7 +212,22 @@ def determineCommitId() {
   }
 }
 
-
+/**
+ * For a given Project, compute a nice dash separated directory name
+ * to store the JUnit XML files in. E.g., Project ":connect:api" -> "connect-api"
```

Comment:
We don't have consistent sub-module naming. The connect-related modules don't have a prefix, while other sub-modules must include a prefix to avoid conflicts to connect's sub modules. This means the helper should only revise the module names for the connect modules.

Reply:
I considered this, but didn't want to add special cases to the "module name finding" method. 

The module name computed here (e.g., `connect-api`, `clients`, `group-coordinator-group-coordinator-api`) will determine the name of the directory in build/junit-xml. As long as its unique, it doesn't matter too much what it is.

Another option here would be to reflect the actual module _paths_ in the output directory paths. So, `:connect:api` test results would end up in "build/junit-xml/connect/api/[test, quarantinedTest]/...". This also cause the test catalog directories to mirror the actual layout of the project.